webpack/components/Errata/index.js

```diff
@@ -13,39 +14,68 @@ import {
   EnhancementIcon,
   SquareIcon,
 } from '@patternfly/react-icons';
-import PropTypes from 'prop-types';
+import { TranslatedAnchor } from '../Table/components/TranslatedPlural';
 
 export const ErrataMapper = ({ data, id }) => data.map(({ x: type, y: count }) => <ErrataSummary count={count} type={type} key={`${count} ${type}`} id={id} />);
 
 export const ErrataSummary = ({ type, count }) => {
   let ErrataIcon;
   let label;
-  let name;
   let url;
   let color;
   switch (type) {
   case 'security':
     label = __('Security');
     ErrataIcon = SecurityIcon;
-    name = 'security advisories';
     color = '#0066cc';
-    url = <a href="#/Content/errata?type=security"> {count} {name} </a>;
+    url = (
+      <TranslatedAnchor
+        id="errata-card-security-count"
+        style={{ marginLeft: '0.4rem' }}
+        href="#/Content/errata?type=security"
+        count={count}
+        plural="security advisories"
+        singular="security advisory"
+        zeroMsg="# security advisories"
+        ariaLabel={`${count} security advisories`}
```

Comment:
React Testing Library won't render the text properly now that it's translated:

```html
<a
      aria-label="0 enhancements"
      href="#/Content/errata?type=enhancement"
      style="margin-left: 0.4rem;"
 >
 {count, plural, =0 {# enhancements} one {# enhancement} other {# enhancements}}
</a>
```
(Looks fine on screen, though.)

Because of this, I had to find another way to locate these elements. I came up with adding an `ariaLabel` prop here which is passed to the `<a>` tag as the `aria-label` HTML attribute. This allows finding the element easily with `getByLabelText`.